### ISSUE TITLE
Installation fails if /etc/firejail/login.users exists

### ISSUE THREAD

**Issue by Fred-Barclay** [netblue30/firejail#3462]
**Bug and expected behavior**
When compiling from source from latest with `./configure --prefix=/usr && make && sudo make install-strip #or install` the process errors  with the message:
```
[ ! -f /etc/firejail/login.users ] && install -m 0644 etc/login.users /etc/firejail
make[1]: *** [Makefile:118: realinstall] Error 1
make[1]: Leaving directory '/home/fred/firejail'
make: *** [Makefile:156: install-strip] Error 2
```

I believe the issue started some point after 378d0e613a915cc0d6d1138565abde2d253af69a, possibly due to dae3933bc30a08e355083b67ee99e688bfa13b4e or the few commits following.

Relevant line seems to be https://github.com/netblue30/firejail/blob/0b237315e6f9fd98af093eda30ab6f555c2db96d/Makefile.in#L125

Not entirely sure why it gives up and dies, not super familiar with makefiles... :confused: 

**Environment**
```
$ sb_release -a
No LSB modules are available.
Distributor ID:	Linuxmint
Description:	LMDE 4 (debbie)
Release:	4
Codename:	debbie

$ cat /etc/debian_version 
10.2
```

**Comment by rusty-snake** [comment]
can you try this:
```
[ ! -f $(DESTDIR)$(sysconfdir)/firejail/login.users ] && install -m 0644 etc/login.users $(DESTDIR)$(sysconfdir)/firejail/login.users
```

**Comment by rusty-snake** [comment]
@Fred-Barclay @glitsj16 ^^

**Comment by glitsj16** [comment]
@rusty-snake For me this is fixed now on Ubuntu 16.04 LTS. Thanks!

**Comment by Fred-Barclay** [comment]
@rusty-snake I haven't been able to check on my Debian box but it seems good now on my Ubuntu 20.04 - thanks! 😄
